src/content/developers/tutorials/how-to-view-nft-in-metamask.md

```diff
@@ -0,0 +1,28 @@
+
+# How to View Your NFT in Your Wallet (Part 3/3 of NFT Tutorial Series)
+
+This tutorial idescribes how to view an existing an NFT on your virtual Ethereum wallet (Metamask in this case)! If you’d like to learn how to mint your own NFT on Ethereum, you should check out Part 1 on How to Write & Deploy an NFT smart contract!
+
+##
+
+Congrats! You’ve made it to the shortest and simplest part of our NFT tutorial series — how to view your freshly minted NFT on a virtual wallet. We’ll be using Metamask for this example since it is what we used in the previous two parts.
+
+As a prerequisite, you should already have Metamask on mobile installed, and it should include the account to which you minted your NFT — you can get the app for free on [iOS](https://apps.apple.com/us/app/metamask-blockchain-wallet/id1438144202) or [Android](https://play.google.com/store/apps/details?id=io.metamask&hl=en_US&gl=US) .
```

Comment:
Open to thoughts on this, but should we just link to [MetaMask homepage](https://metamask.io/) directly and let people download from there?